### PR TITLE
Seek video media event

### PR DESCRIPTION
**Pull request by arslanhashmi**
**Trello Link:** [here](https://trello.com/c/W6S24GlL/68-video-interaction-event-seekvideo-edxvideopositionchanged)

**Description:** A browser emits seek_video events when a user selects a user interface control to go to a different point in the video file.

**Checks before merge:**

- [ ] JSON Reviewed
- [ ] Reviewed
- [ ] Unit Test Pass
- [ ] Commits squashed

### CODE REVIEW

File: openedx/features/caliper_tracking/tests/expected/seek_video.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+    "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+    "action": "Paused",
```
Comment (asamolion) [inline openedx/features/caliper_tracking/tests/expected/seek_video.json]: `Paused` is not a suitable option for this. Please use another appropriate `action`. I believe `JumpedTo` is appropriate in this case.

File: openedx/features/caliper_tracking/transformers/video_transformers.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 import json
 from datetime import timedelta
+
```
Comment (asamolion) [inline openedx/features/caliper_tracking/transformers/video_transformers.py]: Is there any unnecessary whitespace added here?

File: openedx/features/caliper_tracking/tests/expected/seek_video.json (hunk 1 above)
Comment (arslanhashmi) [reply openedx/features/caliper_tracking/tests/expected/seek_video.json]: Oh it was a mistake during updating Json, it is `JumpedTo`.

File: openedx/features/caliper_tracking/transformers/video_transformers.py (hunk 2 above)
Comment (arslanhashmi) [reply openedx/features/caliper_tracking/transformers/video_transformers.py]: There should be a white line between builtin libraries imports and third party packages imports, similarly another line before between current package imports and then similarly with relative imports from repo.

Comment (asamolion) [reply openedx/features/caliper_tracking/transformers/video_transformers.py]: Cool.
